.github/workflows/Create-Custom-Jar.yml

```diff
@@ -15,6 +15,9 @@ jobs:
   create_custom_jar:
     name: Create jar
     runs-on: ubuntu-20.04
+    env:
+      # we use this in env var for conditionals (secrets can't be used in conditionals)
+      AWS_KEY: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
This is used in an if statement on the step that sets up the AWS credentials, if this is missing (or has no value) the credentials are not set.